README.md

```diff
@@ -1 +1 @@
-# Hello-Microverse-CR
\ No newline at end of file
+# Hello-Microverse
```

Comment:
Please, add a descriptive README file. The README file is the first impression a visitor takes about your project, so, it will be nice to make it more descriptive and professional.